bin/cap_vivo_4store

```diff
@@ -28,13 +28,15 @@ case "$1" in
     # Ensure the 4store KB exists for 'cap_vivo'.
```

Comment:
Does this script work without `sudo` access to `/var/lib/4store`?  Does it need to be invoked with `sudo` instead?